proto/rill/runtime/v1/resources.proto

```diff
@@ -350,6 +348,9 @@ message AlertSpec {
   uint32 renotify_after_seconds = 19;
   repeated Notifier notifiers = 21;
   map<string, string> annotations = 20;
+  string resolver = 22;
+  google.protobuf.Struct resolver_properties = 23;
+  google.protobuf.Struct data_properties = 24;
```

Comment:
1. Move `resolver` and `resolver_properties` up where `query_name` was (after `timeout_seconds`) – the field order in code doesn't need to match the field numbers (which are just for wire compatibility).
2. Should be able to remove the `data_properties` field – that's a YAML thing which is parsed into `resolver` and `resolver_properties`